### ISSUE TITLE
Exception on compile:  No such var: find/find-sources-in-dir, compiling:(clojure/tools/namespace/dir.clj:26:17)

### ISSUE THREAD

**Issue by Dexterminator** [untangled-web/untangled-spec#41]
Trying to do lein test-refresh, lein repl etc when requiring untangled-spec suite in my user namespace causes an exception. I don't use or require it anywhere else.
```
(ns user
   (:require
     [untangled-spec.suite :as suite]))
```

```
Exception in thread "main" java.lang.ExceptionInInitializerError
	at clojure.main.<clinit>(main.java:20)
Caused by: java.lang.RuntimeException: No such var: find/find-sources-in-dir, compiling:(clojure/tools/namespace/dir.clj:26:17)
```

My project.clj:
```
(defproject imperimetric "0.1.0-SNAPSHOT"
  :dependencies [[org.clojure/clojure "1.9.0-alpha14"]
                 [org.clojure/clojurescript "1.9.495"]
                 [reagent "0.6.0"]
                 [binaryage/devtools "0.8.3"]
                 [re-frame "0.9.1"]
                 [secretary "1.2.3"]
                 [cljs-ajax "0.5.8"]
                 [compojure "1.5.1"]
                 [instaparse "1.4.3"]
                 [liberator "0.14.1"]
                 [medley "0.8.4"]
                 [frinj "0.2.5"]
                 [cheshire "5.6.3"]
                 [yogthos/config "0.8"]
                 [cljsjs/clipboard "1.5.13-0"]
                 [ring "1.5.0"]
                 [ring/ring-mock "0.3.0"]
                 [org.clojure/tools.namespace "0.3.0-alpha3"]
                 [devcards "0.2.2"]]

  :uberjar-name "imperimetric-standalone.jar"

  :plugins [[lein-cljsbuild "1.1.5"]
            [lein-less "1.7.5"]
            [lein-kibit "0.1.3"]]

  :min-lein-version "2.5.3"

  :source-paths ["src/clj"]

  :clean-targets ^{:protect false} ["resources/public/js/compiled" "target"
                                    "test/js"]

  :figwheel {:css-dirs         ["resources/public/css"]
             :nrepl-host       "0.0.0.0"
             :nrepl-port       7888
             :nrepl-middleware ["cemerick.piggieback/wrap-cljs-repl"]}

  :less {:source-paths ["less"]
         :target-path  "resources/public/css"}

  :hooks [leiningen.less]

  :profiles
  {:dev
   {:dependencies [[pjstadig/humane-test-output "0.8.1"]
                   [figwheel-sidecar "0.5.0"]
                   [com.cemerick/piggieback "0.2.1"]
                   [org.clojure/test.check "0.9.0"]
                   [navis/untangled-spec "1.0.0-alpha3"]]
    :plugins      [[lein-figwheel "0.5.4-3"]
                   [lein-doo "0.1.7"]
                   [com.jakemccrary/lein-test-refresh "0.19.0"]]
    }}

  :uberjar {:aot :all}

  :cljsbuild
  {:builds
   [{:id           "dev"
     :source-paths ["src/cljs"]
     :figwheel     {:on-jsload "imperimetric.core/mount-root"}
     :compiler     {:main                 imperimetric.core
                    :output-to            "resources/public/js/compiled/app.js"
                    :output-dir           "resources/public/js/compiled/out"
                    :asset-path           "js/compiled/out"
                    :source-map-timestamp true}}
    {:id           "min"
     :source-paths ["src/cljs"]
     :jar          true
     :compiler     {:main            imperimetric.core
                    :output-to       "resources/public/js/compiled/app.js"
                    :optimizations   :advanced
                    :closure-defines {goog.DEBUG false}
                    :pretty-print    false}}
    {:id           "devcards"
     :source-paths ["src/devcards" "src/cljs"]
     :figwheel     {:devcards true}
     :compiler     {:main                 imperimetric.core-card
                    :output-to            "resources/public/js/compiled/devcards.js"
                    :output-dir           "resources/public/js/compiled/devcards_out"
                    :asset-path           "js/compiled/devcards_out"
                    :source-map-timestamp true}}
    {:id           "test"
     :source-paths ["src/cljs" "test/cljs"]
     :compiler     {:output-to     "resources/public/js/compiled/test.js"
                    :main          imperimetric.runner
                    :optimizations :none}}
    ]}

  :main imperimetric.server

  :aot [imperimetric.server]

  :prep-tasks [["cljsbuild" "once" "min"] "compile"]
  )


```

Cannot seem to find what the issue is, I'm assuming the issue is not with tools.namespace itself.

**Comment by awkay** [comment]
Definitely a dependencies problem. I'd suggest using 0.5.9 of sidecar. Then use `lein deps :tree` and figure out where you're getting something old.

**Comment by chbrown** [comment]
Not sure if awkay's fix resolved Dexterminator's problem, but I was getting the same "No such var: find/find-sources-in-dir" exception (that had nothing to do with untangled), and a simple `lein clean` fixed it.

```shell
# tl;dr: Inexplicable "No such var: find/find-sources-in-dir" message? Run this:
lein clean
```

This might be off-topic, but Google is weird and brought me here. (Sorry for butting in :/)
